cli.go

```diff
@@ -1,5 +1,87 @@
 package main
 
-func main() {
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+// Command struct is a representation of an isolated command executed by a user
+type Command struct {
+	Command string `json:"Command"`
+	Username string `json:"Username"`
+	Amount string `json:"Amount"`
+	Stock string `json:"Stock"`
+	Filename string `json:"Filename"`
+}
+
+// FromStringToCommandStruct takes a line from the user command file as an input and returns a defined golang structure 
+func FromStringToCommandStruct(line string) *Command {
+	/* 
+		The line variable value should have a similar format to this: '[1] ADD,oY01WVirLr,63511.53' 
+	*/
+	line = strings.Split(line, " ")[1]
+	commandVars := strings.Split(line, ",")
+	cmd := commandVars[0]
+
+	if cmd == "ADD" {
+		return &Command{Command: cmd, Username: commandVars[1], Amount: commandVars[2]}
+	} else if (cmd == "COMMIT_BUY" || cmd == "CANCEL_BUY" || cmd == "COMMIT_SELL" || cmd == "CANCEL_SELL" || cmd == "DISPLAY_SUMMARY") {
```

Comment:
Please remove the else if. Just the ifs are sufficient. Add a new line between each if. This looks unwieldy.

Reply:
You are right.  Technically we don't need the else if's because of the singular return statements. On the 'unwieldy' part, I disagree. But I will change it for the sake of beautification